Keas.Core/Domain/Group.cs

```diff
@@ -18,9 +18,9 @@ public class Group
         public string Name { get; set; }
 
         [JsonIgnore]
-        public List<GroupPermission> GroupPermissions { get; set; }
+        public ICollection<GroupPermission> GroupPermissions { get; set; }
 
         [JsonIgnore]
-        public List<GroupXTeam> Teams { get; set; }
+        public IList<GroupXTeam> Teams { get; set; }

```

Comment:
You are reverting a bunch of changes from master here that I don't want lost.  Can you try pulling in master again or clean up your merge?